src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -275,7 +295,22 @@ private static void addConnectorConfigs(ConfigDef configDef) {
         group, 
         ++order, 
         Width.SHORT, 
-        "Read Timeout");
+        "Read Timeout"
+    ).define(CLIENT_AUTH_CONF,
+        Type.STRING,
+        CLIENT_AUTH_DEFAULT,
+        Importance.MEDIUM,
+        CLIENT_AUTH_DOC,
+        SSL_GROUP,
+        ++order,
+        Width.SHORT,
+        "Client Authentication Required"
+    );
```

Comment:
Follow up comment I forgot. I thought we wanted to add an enum recommender here. `required` is the default. Which are the other options. Probably need a validator too.

Reply:
Actually reading this more closely, I think it should be removed entirely. The connector is always the client, so it is Elastic's responsibility to ask for 2-way SSL. This config only makes sense for brokers, IIUC. Agree? thanks for the catch here, this one was fishy

Reply:
Sounds good removing, since it's not used!